src/Service/ServiceFactory.php

```diff
@@ -223,4 +223,17 @@ public function getVersionsService(): VersionsService
 
         return $this->instances['VersionsService'];
     }
+
+    public function getProgrammesAggregationService(): ProgrammesAggregationService
```

Comment:
Keep the functions in this (any all other) Factories alphabetical so it is easy to find them